nevow/test/test_appserver.py

```diff
@@ -39,18 +58,42 @@ def renderHTTP(self, context):
         return ''
 
 
+def getResourceFor(root, url):
```

Comment:
Can you add some quick docstrings for these functions, explaining their specific purpose?